code/modules/mob/living/carbon/human/species/zombies.dm

```diff
@@ -67,7 +67,7 @@
 		playsound(H, pick(spooks), 50, TRUE, 10)
 
 //Congrats you somehow died so hard you stopped being a zombie
-/datum/species/zombie/infectious/spec_death(gibbed, mob/living/carbon/C)
+/datum/species/zombie/infectious/handle_death(gibbed, mob/living/carbon/C)
```

Comment:
This is actually a bad idea, the reason this works on TG is because zombies don’t die there due to their nodeath trait, but in our code zombies do die and then revive afterwards. I need to revert this back